trans/templates/home.html

```diff
@@ -176,7 +182,7 @@ <h3 style="color:green">
               <select name="extra_country1">
                 <option value="NONE">None</option>
                 {% for user in translating_users %}
-                  <option value="{{ user.country.code }}">{{ user.country.name }} ({{ user.language.code }}_{{ user.country.code2 }})</option>
+                  <option value="{{ user.country.code }}">{{ user.language.name }} ({{ user.language.code }}_{{ user.country.code2 }})</option>

```

Comment:
Need to change the sorting method, now it should sort based on language name, not country name.